index.html

```diff
@@ -9,20 +9,28 @@
         <link rel="stylesheet" href="./style.css">
     </head>
     <body>
-
         <header class="appbar header">
             <div class = "toolbar mobile">
                 <div class="mylogo-mobile mobile">
                     <a href="">My Logo</a>
                 </div>
                 <nav class="menu-container mobile">
-                    <button class="button-burger mobile" href="">
+                    <button class="button-burger mobile">
                         <div class="burger"></div>
                         <div class="burger"></div>
                         <div class="burger"></div>
                     </button>
                 </nav>
             </div>
+            <div class="mobile-menu">
+                <button class="close-button"><img src="./icons/icons_icon-cancel.png" /></button>
+                <ul class="menu-list">
+                    <li><a href="#works" onclick="closeMenu()">Portfolio</a></li>
+                    <li><a href="#about-me" onclick="closeMenu()">About</a></li>
+                    <li><a href="#contact-form" onclick="closeMenu()">Contact</a></li>
+                </ul>
+            </div>
+
             <nav class = "toolbar-d desktop">
```

Comment:
- [ ] I suggest that you check the padding you added to this class that is causing the app not to fit on the mobile device
<img width="1376" alt="Screen Shot 2023-06-26 at 21 07 20" src="https://github.com/AliRezaBmeDu/Portfolio-Website/assets/44745855/f6e0f37d-1663-474d-b930-ce4d1c27be01">

Reply:
Hello @DeliceLydia, thank you for your feedback. 
The class element you marked is for desktop-version and it is not displayed in mobile version. I don't understand how it is conflicting my mobile version. However, I updated my padding and uploaded it again.
Could you please give me a guideline?